teacher.rb

```diff
@@ -0,0 +1,20 @@
+require_relative './person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(age, name, specialization)
+    super(age, name)
```

Comment:
- [ ] [OPTIONAL] Kindly add the `_parental_permission` as a parameter to the initialize so it will be like:
```
def initialize(age, name, _parental_permission, classroom)
```
then pass it to the superclass like:
```
super(age, name, parental_permission: _parental_permission)
```